configs/_base_/datasets/kitti_seg.py

```diff
@@ -0,0 +1,54 @@
+data_root = 'data/kitti-seg/'
+dataset_type = 'KittiSegDataset'
+
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+crop_size = (864, 256)
```

Comment:
I didn't have the reference for this crop size because this database is not very commonly used ... Just randomly selected values and these values are the multiple of 8.

Reply:
This project is top one on the paper with code for this dataset, there are same reference settings, the followed by commit modified this crop size : https://github.com/NVIDIA/semantic-segmentation/blob/7726b144c2cc0b8e09c67eabb78f027efdf3f0fa/train.py#L149-L150

Reply:
Hi, @AkideLiu 
Thanks for your link, I took a quick look at this [repo](https://github.com/NVIDIA/semantic-segmentation/tree/sdcnet).
I found that, the `crop_size` should be set to `360` according to https://github.com/NVIDIA/semantic-segmentation/blob/2f548ab30ab0d56e91de66a4dea4757a0c64e7e4/scripts/train_kitti_WideResNet38.sh#L16.
And in their [paper](https://openaccess.thecvf.com/content_CVPR_2019/papers/Zhu_Improving_Semantic_Segmentation_via_Video_Propagation_and_Label_Relaxation_CVPR_2019_paper.pdf), the `crop_size` on KITTI is set to `368`, you may check it at `4.1 Implementation Details` section.

Besides, they test their model by `slide` mode and `crop_szie` is set to `368` , see [test script](https://github.com/NVIDIA/semantic-segmentation/blob/sdcnet/scripts/test_kitti_WideResNet38.sh).

Reply:
Hi, @xiexinch I have changed the crop size to 368, and changed the test_cfg with the mode to slide.

The stride is calculated manually that ceil(368 * (1-1/3)) = 246,

overlap is not specified in the test script, therefore default one (1/3) has been used. 

https://github.com/NVIDIA/semantic-segmentation/blob/2f548ab30ab0d56e91de66a4dea4757a0c64e7e4/eval.py#L50-L50

I am not sure about the implementation difference of sliding inference between mmseg and Nvidia project, if there are any remaining problems plz point them out.  

https://github.com/NVIDIA/semantic-segmentation/blob/2f548ab30ab0d56e91de66a4dea4757a0c64e7e4/eval.py#L103

https://github.com/AkideLiu/mmsegmentation-Kitti/blob/0f7e95014b86d08e240e27f34349ab382aafe9a8/mmseg/models/segmentors/encoder_decoder.py#L155-L155